app/views/locations/_form.html.erb

```diff
@@ -47,6 +48,8 @@
 
       <br />
       <input editable="false" id="placenameDisplay" type="text" class="form-control" />
+      <br />
+      <button class="btn btn-primary" onclick="addTag('location: blurred')">Save Location</button>
```

Comment:
oops, no space here, should be just `location:blurred` and then we should also use `'lat:' + location.getLat()` and `'lon:' + location.getLat()` -- make sense? That way the button adds all three tags.